hoomd/error.py

```diff
@@ -29,7 +33,7 @@ def __str__(self):
 
 
 class TypeConversionError(ValueError):
-    """Error when validatimg TypeConverter subclasses fails."""
+    """Error when validating TypeConverter subclasses fails."""
```

Comment:
Do we have documentation somewhere on what a `TypeConverter` actually is? If not, this may be confusing to users.

Reply:
We do not. I made the exception documentation more generic.